main.go

```diff
@@ -197,6 +198,8 @@ func main() {
 
 	runner := NewDumpRunner(basePath)
 
-	log.Print("Running tasks...")
-	RunAllTasks(runner, basePath, *concurrentTasks)
+	log.Print("Running dump tasks...")
+	RunAllDumpTasks(runner, basePath, *concurrentTasks)
```

Comment:
do we have to do the dereference here? As in does this have to be a pointer or could it just be a value? Not familiar with all the code so unsure. Again not a big deal